app/blog/models.py

```diff
@@ -23,6 +24,7 @@ def format_date(self) -> str:
     def save(self, *args, **kwargs):
         if not self.slug:
             self.slug = slugify(self.title)
+        cache.clear()
```

Comment:
Be careful, because here you are deleting ALL the cache, not a bid deal, but it would be better if you only deleted the keys related to your blog post views. If you use Redis, you could use a `key_prefix` like `blog-` and then `cache.delete(blog-*)`